rai/node/node.cpp

```diff
@@ -373,11 +370,10 @@ class network_message_visitor : public rai::message_visitor
 		auto vote (node.vote_processor.vote (message_a.vote, sender));
 		if (vote.code == rai::vote_code::replay)
 		{
-			assert (vote.vote->sequence > message_a.vote->sequence);
 			// This tries to assist rep nodes that have lost track of their highest sequence number by replaying our highest known vote back to them
 			// Only do this if the sequence number is significantly different to account for network reordering
 			// Amplify attack considerations: We're sending out a confirm_ack in response to a confirm_ack for no net traffic increase
-			if (vote.vote->sequence - message_a.vote->sequence > 10000)
+			if (vote.vote->sequence + 10000 > message_a.vote->sequence)
```

Comment:
I think this needs to be vote.vote->sequence > message_a.vote->sequence + 10000.

Reply:
Good catch. Fixed.